selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -91,9 +91,21 @@ TogglesPanel::TogglesPanel(SettingsWindow *parent) : ListWidget(parent) {
   std::vector<QString> longi_button_texts{tr("Aggressive"), tr("Standard"), tr("Relaxed")};
   long_personality_setting = new ButtonParamControl("LongitudinalPersonality", tr("Driving Personality"),
                                           tr("Standard is recommended. In aggressive mode, openpilot will follow lead cars closer and be more aggressive with the gas and brake. "
-                                             "In relaxed mode openpilot will stay further away from lead cars."),
+                                             "In relaxed mode openpilot will stay further away from lead cars. On supported cars, you can cycle through these personalities with "
+                                             "your steering wheel distance button."),
                                           "../assets/offroad/icon_speed_limit.png",
                                           longi_button_texts);
+
+  // Watch param to update the personality setting in the UI
+  ParamWatcher *lp_watch = new ParamWatcher(this);
+  lp_watch->addParam("LongitudinalPersonality");
+  QObject::connect(lp_watch, &ParamWatcher::paramChanged, [=](const QString &param_name, const QString &param_value) {
+    lp_watch->addParam("LongitudinalPersonality");
```

Comment:
Haven't checked yet whether we need to do this, saw a comment: `// add these back in case the files got removed`